drafts/25-xxx-specifications.txt

```diff
@@ -28,53 +28,52 @@ discussions, these have included line continuation processing, comment
 handling, and tokenization for the preprocessor.)
 
 
-2. Formal specifications
-========================
+2 Lexical specifications
+------------------------
 
-2.1 Lexical specifications
---------------------------
-
-2.1.1 Lines
------------
+2.1 Lines
+---------
 
 li00. The Fortran preprocessor recognizes three distinct types of lines:
       preprocessing directives (and continuation lines thereof),
       Fortran comment lines, and Fortran source fragments.
 
-li10. A line that has a '#'
+li11. A line that has a '#'
       character as the first non-blank character of the line is a
       directive line (as required by C 2023 6.10.1 paragraph 2),
       except when otherwise specified by the next two rules.
 
-li21. In fixed-form input files, a '#' in column 6 of a non-comment
+li13. In fixed-form input files, a '#' in column 6 of a non-comment
       line does not introduce a directive line.
 
-li22. A preprocessor directive can be continued with a backslash '\'
+li15. A preprocessor directive can be continued with a backslash '\'
       immediately followed by a new-line. The backslash and new-line are
       deleted, the content of the subsequent line are textually appended
       to the current directive, and the subsequent line is deleted.
       This process repeats until the current directive does not end
       with a backslash '\' immediately followed by a new-line.
 
-li23. Preprocessor directive continuation processing described by the prior rule is
+li17. Preprocessor directive continuation processing described by the prior rule is
       effectively performed before any other processing of the text in affected lines.
 
-li23. The maximum length of a preprocessor directive (including
+li19. The maximum length of a preprocessor directive (including
       continuation text) is one million characters.
 
-li24. A non-empty source file shall not end with a '\' followed
-      immediately by a new-line as required in C 2023 5.1.1.2 bullet 2.
+li21. A non-empty source file shall not end with a directive line
+      that ends with a '\' followed
+      immediately by a new-line (as required in C 2023 5.1.1.2 bullet 2).
+
+li31. Fortran comment lines are defined as in 25-007 6.2.2.3 and
+      6.2.3.2.
```

Comment:
The referenced sections don't exist.

I think you mean 6.3.2.3 and 6.3.3.2

Reply:
I believe the technical description for this error is "dammit!"

I don't know how to suggest my own changes to the PR, so feel free to suggest the correction and I will be happy to commit it.

If not, I can update the PR. (I want to fix all your comments...)

Reply:
```suggestion
li31. Fortran comment lines are defined as in 25-007 6.3.2.3 and
      6.3.3.2.
```